eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/phase0/shuffling/ShufflingTestExecutor.java

```diff
@@ -34,14 +34,12 @@
   public void runTest(final TestDefinition testDefinition) throws Exception {
     final ShufflingData shufflingData =
         loadYaml(testDefinition, "mapping.yaml", ShufflingData.class);
-    final CommitteeUtil committeeUtil =
-        testDefinition.getSpec().atSlot(UInt64.ZERO).getCommitteeUtil();
+    final MiscHelpers miscHelpers = testDefinition.getSpec().atSlot(UInt64.ZERO).miscHelpers();
```

Comment:
(nit) There's also `getGenesisSpec`:
```suggestion
    final MiscHelpers miscHelpers = testDefinition.getSpec().getGenesisSpec().miscHelpers();
```